unicorn_contracts/src/contracts_service/updateContractFunction.ts

```diff
@@ -34,103 +26,101 @@ class UpdateContractFunction implements LambdaInterface {
      * @returns {Object} object - API Gateway Lambda Proxy Output Format
      *
      */
-     @tracer.captureLambdaHandler()
-     @metrics.logMetrics()
-     @logger.injectLambdaContext({ logEvent: true })
+    @tracer.captureLambdaHandler()
+    @metrics.logMetrics({ captureColdStartMetric: true })
+    @logger.injectLambdaContext({ logEvent: true })
     public async handler(event: APIGatewayProxyEvent, context: Context): Promise<APIGatewayProxyResult> {
+
+        const modifiedDate = new Date();
+
+        // Assemble from payload
+        let data;
         try {
-            const modifiedDate = new Date();
-
-            // Assemble from payload
-            let data;
-            try {
-                data = await this.validateEvent(event);
-            } catch (error: any) {
-                logger.error(error);
-                const response: APIGatewayProxyResult = {
-                    statusCode: 400,
-                    body: JSON.stringify({ message: (error as Error).message })
-                };
-                tracer.addErrorAsMetadata(error);
-                tracer.addResponseAsMetadata(response, process.env._HANDLER);
```

Comment:
You don't need to call this explicitly, since you are using the decorator this is done for you.